Developer-Guide.md

```diff
@@ -342,7 +342,7 @@ $ curl -L https://raw.githubusercontent.com/kata-containers/packaging/master/ker
 $ make ARCH=${kernel_dir} -j$(nproc)
 $ kata_kernel_dir="/usr/share/kata-containers"
 $ kata_vmlinuz="${kata_kernel_dir}/kata-vmlinuz-${kernel_version}.container"
-$ [ $kernel_arch = ppc64le ] && kernel_file="$(realpath ./vmlinux)" || kernel_file="$(realpath arch/${kernel_arch}/boot/bzImage)"
+$ [ $kernel_arch = ppc64le ] && kernel_file="$(realpath ./vmlinux)" || [ $kernel_arch = aarch64 ] && kernel_file="$(realpath arch/arm64/boot/Image)" || kernel_file="$(realpath arch/${kernel_arch}/boot/bzImage)"
```

Comment:
This is not working as intended, you need a proper `if..elfi...fi` or `case...esac` like:

```
$ case $kernel_arch in; ppc64le) kernel_path="./vmlinux";; aarch64) kernel_path="arch/arm64/boot/Image";; *) kernel_path="arch/${kernel_arch}/boot/bzImage";; esac
$ kernel_file="$(realpath $kernel_path)"
```

Reply:
thanks, updated

Reply:
Thank you!